### PR TITLE
[RU] Update `osu!supporter`

### PR DESCRIPTION
**Pull request by TicClick**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] includes changes from https://github.com/ppy/osu-wiki/pull/7484
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/osu!supporter/ru.md [hunk 1]
```diff
@@ -1,88 +1,162 @@
...
+  - саппортер
+  - осусаппортер
+  - тег
+  - тэг
 ---
 
 # osu!supporter
 
-![Значок саппорта](img/store-product.jpg?1 "Значок саппорта")
+**osu!supporter** (рус. *саппортер*, *саппорт*) — временный статус, а также жаргонное название игроков, которые добровольно пожертвовали деньги на развитие osu!. Саппортер даёт ряд косметических и технических преимуществ, большинство из которых действуют, только пока он активен. Отдельно стоит отметить, что ни одна из возможностей, получаемых от саппортера, не даёт игрового преимущества и не влияет на число получаемых pp или очков.
```
Comment (UristMcKobold) [inline wiki/osu!supporter/ru.md:17]:
```suggestion
**osu!supporter** (рус. *саппортер*, *саппорт*) — временный статус, а также жаргонное название игроков, которые добровольно пожертвовали деньги на развитие osu!. Саппортер даёт ряд косметических и технических преимуществ, большинство из которых действуют только на период его длительности. Отдельно стоит отметить, что ни одна из возможностей, получаемых от саппортера, не даёт и никогда не будет давать преимуществ в игре, вроде числа получаемых pp или очков.
```

File: wiki/osu!supporter/ru.md [hunk 2]
```diff
@@ -1,88 +1,162 @@
...
+![](img/usercard.png?2 "Карточка игрока с саппортером")
+:::
 
-## Процесс покупки
+Каждый саппортер получает значок с сердечками, который обычно показывается около имени пользователя на сайте. Число сердечек зависит от того, как долго пользователь был саппортером (учитывается суммарная длительность подаренных и купленных саппортеров):
 
-### Подсчёт стоимости
+- Менее одного года: одно сердечко
+- От года до пяти лет: два сердечка
+- Больше пяти лет: три сердечка
```
Comment (UristMcKobold) [inline wiki/osu!supporter/ru.md:37]:
```suggestion
- Более пяти лет: три сердечка
```
Менее-более, меньше-больше. У тебя менее-больше.

File: wiki/osu!supporter/ru.md [hunk 3]
```diff
@@ -1,88 +1,162 @@
...
-| 9 | 22 (-39%) |
-| 10 | 24 (-40%) |
-| 12 | 26 (-46% максимум) |
-| 24 | 52 (-46% максимум) |
+::: Infobox
+![](img/chat.png "Саппортер в чате")
+:::
 
-- Скидок на покупку нескольким пользователям нет.
+Ник саппортера во [внутриигровом чате](/wiki/Client/Interface/Chat_console) имеет ярко-жёлтый цвет.
```
Comment (UristMcKobold) [inline wiki/osu!supporter/ru.md:45]:
```suggestion
Ник саппортера во [внутриигровом чате](/wiki/Client/Interface/Chat_console) отображается ярко-жёлтым цветом для других игроков.
```

File: wiki/osu!supporter/ru.md [hunk 4]
```diff
@@ -1,88 +1,162 @@
...
+*Примечание: если вы хотите незначительно изменить ник, то техподдержку можно бесплатно попросить и без покупки саппорта — см. [Справочный центр/Аккаунт § Можно ли поменять ник?](/wiki/Help_centre/Account#можно-ли-поменять-ник?).*
 
-## Проблемы
+После покупки саппортера можно один раз бесплатно поменять ник [на обычных условиях](wiki/Help_centre/Account#смена-ника).
 
-### Я купил(а) саппорт, но в игре ничего не поменялось!
+### osu!direct
 
-Проверьте профиль: появилось ли там уведомление? Если да, перезапустите клиент игры.
+osu!direct — это инструмент для поиска и скачивания карт, встроенный в игру и доступный только саппортерам. Попасть в него можно из главного меню, нажав на огромную вертикальную кнопку с надписью `osu!direct`, которая появится справа. Помимо внутриигрового поиска карт, osu!direct добавляет ещё несколько возможностей для скачивания карт без покидания osu!:
```
Comment (UristMcKobold) [inline wiki/osu!supporter/ru.md:74]:
```suggestion
osu!direct — это инструмент для поиска и скачивания карт, встроенный в игру и доступный только саппортерам. Попасть в него можно из главного меню, нажав на огромную вертикальную кнопку с надписью `osu!direct`, которая появится справа. Помимо внутриигрового поиска карт, osu!direct добавляет ещё несколько возможностей для скачивания карт без сворачивания osu!:
```

**Review by cl8n — APPROVED** [review]

**Comment by cl8n** [comment]
will leave it to you to update this translation appropriately again depending on if 7484 gets more updates

**Pull request merged**